src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -183,6 +183,8 @@ private function getPathOperation(string $operationName, array $operation, strin
                 return $this->updateGetOperation($pathOperation, $mimeTypes, $operationType, $resourceMetadata, $resourceClass, $resourceShortName, $operationName, $definitions);
             case 'POST':
                 return $this->updatePostOperation($pathOperation, $mimeTypes, $operationType, $resourceMetadata, $resourceClass, $resourceShortName, $operationName, $definitions);
+            case 'PATCH':
+                $pathOperation['summary'] ?? $pathOperation['summary'] = sprintf('Updates the %s resource.', $resourceShortName);
```

Comment:
interesting hack :stuck_out_tongue: we could `break` after this IMO so that it gets to the return statement directly.